fl4health/clients/basic_client.py

```diff
@@ -1105,6 +1151,29 @@ def get_model(self, config: Config) -> nn.Module:
         """
         raise NotImplementedError
 
+    def get_lr_scheduler(self, optimizer_key: str, config: Config) -> Union[None, _LRScheduler]:
+        """
+        Optional user defined method that returns learning rate scheduler
+        to be used throughout training for the given optimizer. Defaults to None.
+
+        Args:
+            optimizer_key (str): The key in the optimizer dict corresponding
+                to the optimizer we are optionally defining a learning rate
+                scheduler for.
+            config (Config): The config from the server.
+
+        Returns:
+            Union[None, _LRScheduler]: Client learning rate schedulers.
+        """
+        return None
+
+    def update_lr_schedulers(self) -> None:
```

Comment:
Might be useful to have access to state information here such as the current epoch and step. However i know the upcoming basic client refactor will be addressing this to some extent. Also might be useful to mention that self.total_steps is available in the docstring